### DISCUSSION TITLE
Grammarly on Desktop by passing SEB Controls

### DISCUSSION THREAD

**Discussion by nvarghese-rm** [SafeExamBrowser/seb-win-refactoring#507] [Q&A]
Hi 

We have a situation that the local desktop installation of Grammarly is bypassing SEB controls and displaying the Grammarly check within SEB browser in v3.0. 

Is there any reason why this is occurring now when it didn't use to in v2.0?

Would the settings 'Ignore SEB Service' and 'Kiosk Mode' affect this?

Thanks

Neal

**Comment by dbuechel** [comment]
Yes, the kiosk mode may affect the behavior of third-party applications, see also https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane. In general, we recommend using Create New Desktop, the default and more secure kiosk mode. In case Grammarly should still work, you can simply add it to the list of prohibited applications in your configuration, then SEB should terminate it during startup: https://safeexambrowser.org/windows/win_usermanual_en.html#ProhibitedProcessesSection.

**Reply by nvarghese-rm** [reply]
Thanks for the prompt response. This would not have been any different in SEB v2.0, I assume. 

Also are you also saying 'Ignore SEB Service' would not make a difference to this and is only restricting OS features.

**Reply by dbuechel** [reply]
> Also are you also saying 'Ignore SEB Service' would not make a difference to this and is only restricting OS features.

Yes exactly, the SEB Service would not have any effect on this issue.

**Reply by nvarghese-rm** [reply]
Was kiosk mode operation the same in SEB v2.0 as in v3.0?

**Reply by dbuechel** [reply]
Yes, the kiosk mode was implemented roughly the same way (via the same OS mechanisms). However, the application monitoring has been rewritten, albeit based on the old code. Furthermore we're now using a different browser engine (Chromium instead of Firefox), so this might indeed also have an influence in the behavior you described.
